cmd/syncthing/gui.go

```diff
@@ -872,24 +843,21 @@ func (s *apiSvc) getSystemUpgrade(w http.ResponseWriter, r *http.Request) {
 	res["newer"] = upgrade.CompareVersions(rel.Tag, Version) == upgrade.Newer
 	res["majorNewer"] = upgrade.CompareVersions(rel.Tag, Version) == upgrade.MajorNewer
 
-	w.Header().Set("Content-Type", "application/json; charset=utf-8")
-	json.NewEncoder(w).Encode(res)
+	sendJSON(w, res)
 }
 
 func (s *apiSvc) getDeviceID(w http.ResponseWriter, r *http.Request) {
 	qs := r.URL.Query()
 	idStr := qs.Get("id")
 	id, err := protocol.DeviceIDFromString(idStr)
-	w.Header().Set("Content-Type", "application/json; charset=utf-8")
+
+	var info = map[string]string{}
```

Comment:
Nitpicking. This probably could have two sendJson calls, one in each branch, or we should use info := make(map...) to make it more consistant with the rest of code, or just removing the equals sign (not sure if that will work without a make).